cmd/lotus-miner/info.go

```diff
@@ -292,7 +292,10 @@ func handleMiningInfo(ctx context.Context, cctx *cli.Context, fullapi v0api.Full
 	if err != nil {
 		return xerrors.Errorf("getting available balance: %w", err)
 	}
-	spendable = big.Add(spendable, availBalance)
+
+	if availBalance.GreaterThan(big.Zero()) {
+		spendable = big.Add(spendable, availBalance)
+	}
```

Comment:
I don't think this is the right spot to make this change. We still want to display a negative miner balance if it exists, we just don't want to add a negative miner balance to the worker balance on line 322.

Reply:
this doesnt impact the miner balance tho?
spendable is only printed out on line 337 - the total balance spendable

Reply:
Oh, you're right, this does work. I just hate the way it was originally coded